include/z64.h

```diff
@@ -1019,14 +1021,25 @@ typedef struct LoadedParticleEntry
     /* 0x005F */ u8 type;
 } LoadedParticleEntry; // size = 0x60
 
+// Some animation related structure
+typedef struct
+{
+    /* 0x00 */ u32      animation;
+    /* 0x04 */ f32      playbackSpeed;
+    /* 0x08 */ f32      unk_08;
+    /* 0x0C */ f32      frameCount;
+    /* 0x10 */ u8       unk_10;
+    /* 0x14 */ f32      transitionRate;
+} struct_80034EC0_Entry; // size = 0x18
+
 typedef struct
 {
     /* 0x00 */ u8 limbCount;
     /* 0x01 */ char unk_01[0x01];
     /* 0x02 */ u8 dListCount;
     /* 0x03 */ char unk_03[0x01]; /* Probably Padding */
     /* 0x04 */ u32 limbIndex;
-    /* 0x08 */ u32 animCurrent;
+    /* 0x08 */ u32* animCurrent;
```

Comment:
Not sure if we have a convention here, but should this actually be UNK_PTR for now?

Reply:
as well as at least one other u32* in the c file